source/javascripts/controllers/_CodeSigningController.js.erb

```diff
@@ -7,13 +7,16 @@ angular.module("BitriseWorkflowEditor").controller("CodeSigningController", func
 	var viewModel = this;
 
 	viewModel.shouldShowCodeSigningHints;
+	viewModel.shouldShowSectionOfCodeSigningHints;
 	viewModel.loadProgress = new Progress();
 
 	function load() {
 		viewModel.loadProgress.start("<%= data[:strings][:code_signing][:available_project_types_load] %>");
 
 		appService.getAvailableProjectTypes().then(function() {
-			viewModel.shouldShowCodeSigningHints = $cookies.get("code_signing_hints_hidden") ? false : viewModel.areAnyOfTheProjectTypesAvailable(["ios", "osx", "macos", "xamarin"]);
+			viewModel.shouldShowSectionOfCodeSigningHints = viewModel.areAnyOfTheProjectTypesAvailable(["ios", "osx", "macos", "xamarin"]);
```

Comment:
Leave it as it is. And for the other variable, call it `viewModel.isCodeSigningHintsOpen`.